prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -28,26 +34,24 @@ import java.time.OffsetDateTime
 import java.util.UUID
 import kotlin.test.Test
 import kotlin.test.assertEquals
-import kotlin.test.assertFails
 import kotlin.test.assertNotNull
-import kotlin.test.assertNull
 import kotlin.test.assertTrue
 
```

Comment:
Remove the `kotling.test` references please.

Reply:
Agree.